fw-child/resources/js/child-functions.js

```diff
@@ -708,6 +709,25 @@ const $ = jQuery;
 
     };
 
+    //
+    // Dynamic logo block display scroll behavior
+    //
+    //
+
+    if ($('#floating-header').length && $('#hero').length) {
+      $(window).scroll(function() {
```

Comment:
Please use jQuery .on
```suggestion
      $( window ).on( 'scroll', function() {
```